scss/components/_soe_news.scss

```diff
@@ -99,11 +99,19 @@
   }
 }
 
+#block-ds-extras-banner {
+  margin-bottom: -70px;
+
+  @media (max-width: 485px) {
```

Comment:
Is there a breakpoint with this pixel dimension? Shouldn't it use one of the mixins?

eg:
```
@include breakpoint-max(med-small) {
    margin-bottom: 0;
}
```

Reply:
I don't think there is. Which is why it looking janky.